hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -484,7 +487,7 @@ def main():
     insar_sentinel_gamma(args.reference, args.secondary, rlooks=args.rlooks, alooks=args.alooks,
                          include_look_vectors=args.l, include_displacement_maps=args.s,
                          include_wrapped_phase=args.w, include_inc_map=args.i,
-                         include_dem=args.d, apply_water_mask=args.m)
+                         include_dem=args.d, apply_water_mask=args.m, phase_filter_thrshold=args.t)
```

Comment:
```suggestion
                         include_dem=args.d, apply_water_mask=args.m, phase_filter_thrshold=args.phase_filter_threshold)
```
Argparse will use the long name if provided.